faqs.mdx

```diff
@@ -1,5 +1,53 @@
 ---
 title: 'FAQs'
-description: 'Documentation for Faqs'
+description: 'Answers to your most common questions about Amplication usage, management, and implementation'
 icon: 'question'
----
\ No newline at end of file
+---
+
+Here are some of the most common questions we receive about Amplication. If you don't find the answer you're looking for, please reach out to us on [Discord](https://amplication.com/discord) or [contact our team](https://amplication.com/contact-us) directly.
+
+<AccordionGroup>
+<Accordion title="What's the difference between Blueprints and Plugins?">
+Blueprints are like architectural drawings that define what you want to build - they specify the type of resource, its properties, and relationships with other blueprints. Think of them as templates that standardize what you build across your organization.
+
+Plugins, on the other hand, contain the actual code that determines how those specifications are implemented. They handle the concrete details like code generation, infrastructure setup, and deployment processes. You need both: Blueprints for standardization and Plugins for implementation.
+
+<Note>Learn more about implementing custom solutions in our [Private Plugins guide](day-zero/private-plugins).</Note>
+</Accordion>
+
+<Accordion title="When should I use custom Blueprints vs. Amplication's standard Blueprints?">
+Amplication offers their own standards through Node.js and .NET. They are ideal if you quickly want to adopt industry best practices and leverage Amplication's public plugin ecosystem. They offer faster onboarding and immediate access to proven patterns.
+
+Custom Blueprints are better when you have specific organizational requirements, technology mandates, or unique compliance needs. They offer you complete control of your service architecture but require more initial setup.
```

Comment:
They ensure you can generate the code based on your organization's own standards and requirements.